packages/arb-token-bridge-ui/src/components/TransferPanel/USDCWithdrawal/USDCWithdrawalConfirmationDialog.tsx

```diff
@@ -40,12 +40,12 @@ export function USDCWithdrawalConfirmationDialog(
     name: USDCFastBridge.name,
     imageSrc: USDCFastBridge.imageSrc,
     href: USDCFastBridge.getHref({
-      from: from.id,
-      to: to.id,
-      fromTokenAddress: isArbitrumGoerli
+      from: childChain.id,
+      to: parentChain.id,
+      fromTokenAddress: isTestnet
```

Comment:
i think we should keep `isArbitrumGoerli` rather than `isTestnet` because we'll add CCTP sepolia support soon and goerli would stay here for a bit longer too